package.json

```diff
@@ -15,10 +14,7 @@
     "storybook": "start-storybook -p 9009 --ci",
     "dev": "yarn storybook",
     "build-storybook": "build-storybook",
-    "build": "yarn build:config && yarn build:packages && yarn build:cleanup",
-    "build:config": "mv tsconfig.json tsconfig.tmp.json && mv tsconfig.production.json tsconfig.json",
```

Comment:
Unfortunately I think we still need this `tsconfig.json` dance because of https://github.com/parcel-bundler/parcel/issues/1251#issuecomment-701546994

I've just tested this PR locally and the issue is happening again:

![image](https://user-images.githubusercontent.com/175330/104181197-417a4080-5406-11eb-8dd3-05119f751d72.png)

A bit frustrating that we have to add this workaround for now 🙁

Reply:
Ah bummer, good catch though. I'll revert this and add a comment somewhere to note for future reference.